### ISSUE TITLE
Lint checks not passing on master

### ISSUE THREAD

**Issue by gdurif** [getkeops/keops#161]
See https://github.com/getkeops/keops/pull/160/checks

`ref_jmlr` was branched from master, and the files concerned by the failing checks are not modified by PR #160 so there seems to be an issue on master regarding these files.

**Comment by gdurif** [comment]
Here is the failed check result:
```
Run psf/black@stable
  env:
    pythonLocation: /opt/hostedtoolcache/Python/3.9.4/x64
    LD_LIBRARY_PATH: /opt/hostedtoolcache/Python/3.9.4/x64/lib
/usr/bin/docker run --name e4d503d636832a4ccdbe4b2ad24f097b50_814616 --label 5588e4 --workdir /github/workspace --rm -e pythonLocation -e LD_LIBRARY_PATH -e HOME -e GITHUB_JOB -e GITHUB_REF -e GITHUB_SHA -e GITHUB_REPOSITORY -e GITHUB_REPOSITORY_OWNER -e GITHUB_RUN_ID -e GITHUB_RUN_NUMBER -e GITHUB_RETENTION_DAYS -e GITHUB_ACTOR -e GITHUB_WORKFLOW -e GITHUB_HEAD_REF -e GITHUB_BASE_REF -e GITHUB_EVENT_NAME -e GITHUB_SERVER_URL -e GITHUB_API_URL -e GITHUB_GRAPHQL_URL -e GITHUB_WORKSPACE -e GITHUB_ACTION -e GITHUB_EVENT_PATH -e GITHUB_ACTION_REPOSITORY -e GITHUB_ACTION_REF -e GITHUB_PATH -e GITHUB_ENV -e RUNNER_OS -e RUNNER_TOOL_CACHE -e RUNNER_TEMP -e RUNNER_WORKSPACE -e ACTIONS_RUNTIME_URL -e ACTIONS_RUNTIME_TOKEN -e ACTIONS_CACHE_URL -e GITHUB_ACTIONS=true -e CI=true -v "/var/run/docker.sock":"/var/run/docker.sock" -v "/home/runner/work/_temp/_github_home":"/github/home" -v "/home/runner/work/_temp/_github_workflow":"/github/workflow" -v "/home/runner/work/_temp/_runner_file_commands":"/github/file_commands" -v "/home/runner/work/keops/keops":"/github/workspace" 5588e4:d503d636832a4ccdbe4b2ad24f097b50
--- pykeops/common/lazy_tensor.py	2021-04-08 08:04:13.342171 +0000
+++ pykeops/common/lazy_tensor.py	2021-04-08 08:04:17.354406 +0000
@@ -2433,13 +2433,17 @@
         if other._shape[-1] == 1:
             return other.binary(self, "ComplexRealScal", **kwargs, is_complex=True)
         elif not is_complex_lazytensor(other):
             return self.mulop(other.real2complex())
         elif self._shape[-1] == 2:
-            return self.binary(other, "ComplexScal", **kwargs, is_complex=True, dimcheck=None)
+            return self.binary(
+                other, "ComplexScal", **kwargs, is_complex=True, dimcheck=None
+            )
         elif other._shape[-1] == 2:
-            return other.binary(self, "ComplexScal", **kwargs, is_complex=True, dimcheck=None)
+            return other.binary(
+                self, "ComplexScal", **kwargs, is_complex=True, dimcheck=None
+            )
         else:
             return self.binary(other, "ComplexMult", **kwargs, is_complex=True)
 
     def addop(self, other, **kwargs):
         if not is_complex_lazytensor(other):
would reformat pykeops/common/lazy_tensor.py
Oh no! 💥 💔 💥
1 file would be reformatted, 114 files would be left unchanged.
```

**Comment by jeanfeydy** [comment]
Fixed with 584da408dd83ffca23c66002c8264f6aa5703ea4.
